hudi-hadoop-mr/pom.xml

```diff
@@ -125,6 +125,10 @@
       <artifactId>mockito-junit-jupiter</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-junit-jupiter</artifactId>
```

Comment:
Scope: test? Or is that inherited

Reply:
It was inherited but made it explicit

Reply:
this lib is already added, just above these lines :)

Reply:
yes.. lets remove this.. may be a rebase/merge thing?

Reply:
done